service/telemetry/config.go

```diff
@@ -83,6 +84,19 @@ type LogsConfig struct {
 	//
 	// By default, there is no initial field.
 	InitialFields map[string]any `mapstructure:"initial_fields"`
+
+	// Rotation sets the way the log is rotated.
+	// Example:
+	//
+	//     rotation:
+	//         max_megabytes: 100
+	//         max_days: 30
+	//         max_backups: 100
+	//         localtime: false
+	//
+	// By default, max size is 100MB before rotation. Max number of backups is 100,
+	// and no limit for days. UTC time will be used.
```

Comment:
Is this already documented in the config struct? I'd avoid duplication here and remove those 2 lines.

Reply:
also by default on shipping, I'd make sure enabled is off by default.

Reply:
I'll update these. Thanks

Reply:
Updated in latest commit.